### PR TITLE
Support for multiple BLS keys

### PR DESCRIPTION
**Pull request by manish0338**
Changes: 
1. Added new multiKey struct
2. Added wrapper functions for Multikey support
3. Handled all the cascading errors changes

Tests : 
1. Tests that run with deploy.sh  
Result: Pass
```
====== WALLET BALANCES ======
Using local profile for wallet
Account: one1spshr72utf6rwxseaz339j09ed8p6f8ke370zj:
    Balance in Shard 0:  54.714075714, nonce: 10 
    Balance in Shard 1:  0.0000, nonce: 0 
Using local profile for wallet
Account: one1uyshu2jgv8w465yc8kkny36thlt2wvel89tcmg:
    Balance in Shard 0:  0.0000, nonce: 0 
    Balance in Shard 1:  63.38074238, nonce: 10 
Using local profile for wallet
Account: one1r4zyyjqrulf935a479sgqlpa78kz7zlcg2jfen:
    Balance in Shard 0:  1.0000, nonce: 0 
    Balance in Shard 1:  1.0000, nonce: 0 
====== RESULTS ======
PASS
```
2. Checked leader change by manually killing leader
Result: Pass

### CODE REVIEW

**Comment by harmony-ek** [comment]
Jenkins speaking; can one of the admins verify this patch?

**Comment by Daniel-VDM** [comment]
Jenkins, test this please

File: cmd/harmony/main.go [hunk 1]
```diff
@@ -220,21 +222,56 @@ func setupInitialAccount() (isLeader bool) {
...
+		nodeConfig.ConsensusPriKey = nodeconfig.AppendPriKey(nodeConfig.ConsensusPriKey, consensusPriKey)
+		nodeConfig.ConsensusPubKey = nodeconfig.AppendPubKey(nodeConfig.ConsensusPubKey, consensusPriKey.GetPublicKey())
+		if len(nodeConfig.ConsensusPriKey.PrivateKey) == 0 || len(nodeConfig.ConsensusPubKey.PublicKey) == 0 {
+			fmt.Println("error to get consensus keys.")
+			os.Exit(100)
+		}
 	}
-	return pubKey
+
+	return nodeConfig.ConsensusPubKey.PublicKey[0]
```
[the remark this answers is not in the document]
Comment (manish0338) [reply cmd/harmony/main.go]: Key returned is used to setup initialAccount.
I am assuming all keys are pointing to same one address.

File: cmd/harmony/main.go [hunk 2]
```diff
@@ -220,21 +222,56 @@ func setupInitialAccount() (isLeader bool) {
...
+			nodeConfig.ConsensusPriKey = nodeconfig.AppendPriKey(nodeConfig.ConsensusPriKey, consensusPriKey)
+			nodeConfig.ConsensusPubKey = nodeconfig.AppendPubKey(nodeConfig.ConsensusPubKey, consensusPriKey.GetPublicKey())
+		}
+	}
+
+	return nil
+}
+
+func setupConsensusKey(nodeConfig *nodeconfig.ConfigType) *bls.PublicKey {
+	if stat, err := os.Stat("./MultiKeys"); err == nil && stat.IsDir() {
```
[the remark this answers is not in the document]
Comment (manish0338) [reply cmd/harmony/main.go]: okay

**Comment by fxfactorial** [comment]
@gupadhyaya please take note of this PR for inspiration.

**Pull request closed without merging**